### DISCUSSION TITLE
released on this day/this week/anniversary

### DISCUSSION THREAD

**Discussion by droopified** [meisnate12/Plex-Meta-Manager#1106] [Ideas]
i just came across this and am not sure if this feature already exists but i have been asking for this on plex and kodi but nobody has taken on the task. 

would it be possible to have a collection that is updated either daily or weekly that have all the movies released on this day or this week? or possibly all anniversaries of increments of 5 years?

**Comment by chazlarson** [comment]
Interesting Idea, but it seems expensive to implement, since most everything that does these sort of searches wants a range of dates or a specific date, not a list of arbitrary dates, so this will involve a lot of searches.  

Also maybe some logic to figure out what "this week" was in some previous year, depending on whether you want "this week 4 years ago" to mean October 10-16 [Wed-Tues] or Oct 7-13 [second week of October 2018] or something else.
